.circleci/config.yml

```diff
@@ -10,22 +12,59 @@ jobs:
           command: |
             sudo pip install --upgrade pre-commit==2.15.0
             pre-commit run --all-files
+  test-and-build:
+    docker:
+      - image: circleci/python:3.9-buster
```

Comment:
Maybe use the `3.9-bullseye` tag, which seems closer to what's used inside the Docker image? Probably doesn't make much of a difference.